guides/discordBot.md

```diff
@@ -2,14 +2,15 @@
 In order to use this bot in Discord, you have to create a bot account and invite it to your server. This guide will show you how to do that.
 
 1. Go to the [Discord Developer Portal](https://discord.com/developers/applications) and click `New Application`.
-2. Enter a name for your bot, like `AutoRSA` and click `Create`.
-3. Click on `Bot` in the left sidebar, then give it a name and profile picture if you want.
-4. Disable `Public Bot`.
-5. Under `Privileged Gateway Intents`, enable `Message Content Intent`.
-6. Click on `OAuth2` in the left sidebar, then `URL Generator`. Then scroll down to `OAuth2 URL Generator`. 
-7. Under `Scopes` select `bot`. Then underneath that in `Bot Permissions` select `Send Messages` and `Read Message History`.
-8. Copy the link in the `Scopes` section and paste it into your browser. Select the server you want to add the bot to and click `Authorize`.
-9. Click on `Bot` in the left sidebar. Under `Token`, click `Reset Token`. Copy the new token and paste it into your `.env` file as `DISCORD_TOKEN`.
-10. To get the Channel ID, go to `Advanced` in Discord settings, then turn on `Developer Mode`. Then right click on the channel for the bot and click `Copy ID`. Paste the ID into your `.env` file as `DISCORD_CHANNEL`. If you want to turn off `Developer Mode`, you can do so, but it isn't necessary.
+2. Enter a name for your bot, like `AutoRSA` and click `Create.
```

Comment:
This line appears unchanged (except for the missing tick on create). Can you put that back?

Reply:
@fewtime Do you still want me to review this?

Reply:
Sorry, I almost forgot this PR and I am not very familiar with Github modifying the file directly.
I have put the tick back on `Create` and I want you to review it if you're free. Thanks.